web/angular-wallet/angular.json

```diff
@@ -58,6 +58,35 @@
                                 }
                             ]
                         },
+                        "desktop": {
+                            "fileReplacements": [
+                                {
+                                    "replace": "src/environments/environment.ts",
+                                    "with": "src/environments/environment.prod.ts"
+                                },
+
+                                {
+                                    "replace": "src/app/ledger/ledger.service.ts",
+                                    "with": "src/app/ledger/ledger.service.desktop.ts"
```

Comment:
Here is where the magic happens. We replace the web version with the desktop version when `--configuration=desktop` flag is provided to angular-cli.